ui/packages/catalog-ui-search/src/main/webapp/component/search-form/search-form.view.js

```diff
@@ -55,8 +56,38 @@
                 this.options.queryModel.set('type', 'text');
                 user.getQuerySettings().set('type', 'text');
                 break;
+            case 'newResult':
+                oldType = this.options.queryModel.get('type');
+                this.options.queryModel.set({
+                    type: 'newResult',
+                    title: "",
+                    modelId: "",
+                    accessGroups: [],
+                    accessIndividuals: [],
+                    descriptors: [],
+                    description: ""
+                });
+                if (oldType  === 'newResult') {
+                    this.options.queryModel.trigger('change:type');
+                }
+                break;              
+            case 'result':
+                oldType = this.options.queryModel.get('type');
+                this.options.queryModel.set({
```

Comment:
Instead of setting an object here, change it to a new instantiation of SearchForm model so we don't keep around old data.